backend/sftp/sftp.go

```diff
@@ -82,6 +82,9 @@ func init() {
 Only PEM encrypted key files (old OpenSSH format) are supported. Encrypted keys
 in the new OpenSSH format can't be used.`,
 			IsPassword: true,
+		}, {
+			Name: "pubkey_file",
+			Help: "Optional path to public key file; set this if you have a signed certificate you want to use for authentication." + env.ShellExpandHelp,
```

Comment:
Can you split this over multiple lines? The first line is then used for the command line help - we don't want this to be too long, so something like

```go
Help: `Optional path to public key file.

Set this if you have a signed certificate you want to use for authentication.` + env.ShellExpandHelp,
```

Also, for discussion - should we set the `Advanced: true` flag here to push this into the advanced config in the config wizard?

Reply:
I thought about this, but I don't think this should Advanced.  If a user has a cert then it's a simpler work flow to configure it straight off, and if they don't then they can just hit return to ignore.